### ISSUE TITLE
Lighthouse report

### ISSUE THREAD

**Issue by naveed-ahmad** [quran/quran.com-frontend#897]
Here is lighthouse report of current app: https://googlechrome.github.io/lighthouse/viewer/?gist=c66b549e98d9931bf477b06dcac5d321

There is a lot of room for improvements.

**Comment by mmahalwy** [comment]
@naveed-ahmad I agree. Also, from a place where we are encompassing everyone and welcoming everyone, we should do a better job on accessibility. We left that one slide unfortunately.
